scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -309,14 +327,17 @@ function Initialize-IoTEdge {
     Setup-Environment -ContainerOs $ContainerOs -SkipArchCheck -SkipBatteryCheck
 
     $configPath = Join-Path -Path $EdgeDataDirectory -ChildPath 'config.yaml'
-    if (Test-Path $configPath) {
+    if ((-not $Force) -and (Test-Path $configPath)) {
         Write-HostRed
         Write-HostRed "$configPath already exists."
         Write-HostRed ('Delete it using "Uninstall-IoTEdge -Force" and then ' +
             're-run "Deploy-IoTEdge" and "Initialize-IoTEdge"')
         throw
     }
 
+    New-Sockets $EdgeDataDirectory
+    Set-SystemPath
```

Comment:
This was removed from here intentionally. Why are you adding it back?

Reply:
multiple customers are reporting this to us as a bug.    'iotedge check' and docker cli don't work after install.  why was this removed?

Reply:
https://github.com/Azure/iotedge/commit/87957c9a92b4df2a7fb5e77951e60184bc94e6a2

>Add the IoT Edge and Moby install directories to PATH in `Deploy-IoTEdge`
>rather than in `Initialize-IoTEdge`. On IoT Core both `Deploy-` and updating
>PATH require a reboot, so it's good to combine those reboots into one.
>
>Fixes #1404

Reply:
but, for manufacturing flows that don't use deploy in the script then they don't get the path set ever.  presumably we could check in initialize and only add to the path if it isn't already there. but, i'd rather treat that as a separate new work item and get this PR completed.

Reply:
Okay, if "manufacturing flows" don't use `Deploy-` then yes it needs to be in both.

>presumably we could check in initialize and only add to the path if it isn't already there

`Set-SystemPath` does that already.